src/insights_client/metrics.py

```diff
@@ -1,9 +1,15 @@
 import os.path
 import re
-
+import sys
+import logging
 import requests
 from six.moves import configparser
+from insights.client.constants import InsightsConstants
+# from insights.client.config import InsightsConfig
```

Comment:
Is it safe to import from the core? I remember @subpop requesting to find a way to import from the bundled egg. The justification was that the metrics should work even if there are bugs in the automatically updated core.

Reply:
Removing this dependency.